.github/workflows/labeler.yml

```diff
@@ -0,0 +1,11 @@
+name: "Pull Request Labeler"
+on:
+- pull_request_target
+
```

Comment:
Need to add

```
permissions:
  contents: read
  pull-requests: write
```

so the auto-generated token can set labels on the PRs.

Reply:
first time setting up GHA actions, where did you see this ? it's not shown in the example https://github.com/marketplace/actions/labeler#create-workflow